.gitignore

```diff
@@ -5,4 +5,6 @@
 static
 .coverage
 coverage.xml
-.env
\ No newline at end of file
+.env
+docker-compose.override.yml
+backup
```

Comment:
To follow the convention of our other repos, we actually want the empty backup directory in version control. To do this, you can create the directory and place a .gitignore file inside. eg.
https://github.com/WikipediaLibrary/externallinks/blob/master/backup/.gitignore

This may seem nitpicky, but it decouples the management of that directory from the docker toolchain which has historically made unannounced breaking changes and does not always behave identically between docker compose and docker swarm.

Reply:
Got it, that totally makes sense!